src/services/QueueManager.ts

```diff
@@ -0,0 +1,156 @@
+import { ConfigData } from '@/configuration';
+import Bull, { Queue, ProcessPromiseFunction, JobOptions, JobCounts } from 'bull';
+import logger from '@/helper/logger';
+import { PlatformMessage } from '@/interfaces/PlatformMessage';
+import { RedisOptions } from 'ioredis';
+
+export type JobData = {
+	serviceType: string;
+	platformId: string;
+	message: PlatformMessage;
+	receiver: string;
+	messageId?: string;
+};
+
+export default class QueueManager {
+	private queues: Queue[] = [];
+
+	/**
+	 *
+	 * @param serviceType
+	 * @param platformId
+	 * @param config the queue config
+	 */
+	createQueue(serviceType: string, platformId: string, config: ConfigData): Queue {
+		const name = this.getQueueName(serviceType, platformId);
+
+		logger.debug(`[queue] initialize service queue ${name} with config:\n`, config);
```

Comment:
util.inspect(config) is not required?

Reply:
I'm not quite sure how we want to format objects in the log output. The default logger does inline formatting. util.inspect would do pretty formatting. Which one is preferable? Currently we have both but I think we have to be consistent.